CHANGELOG.md

```diff
@@ -21,6 +21,10 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 - Add the ``configureBatchActions`` method to the AdminExtensionInterface
 - Added the `getAccessMapping` method to the AdminExtensionInterface
 
+### Changed
+- Changed `AbstractAdmin::configureActionButtons` method signature
+- Moved default buttons form `AbstractAdmin::configureActionButtons` to `AbstractAdmin::configureActionButtons`
```

Comment:
From `configureActionButtons` to `configureActionButtons`? Bad copy paste? :trollface: